pages/index.vue

```diff
@@ -1,26 +1,258 @@
 <template>
-  <section class="container">
-    <div>
-      <logo />
-      <h1 class="title">
-        akira
-      </h1>
-      <h2 class="subtitle">
-        The Linux Design Tool
-      </h2>
-      <div class="links">
-        <a href="https://nuxtjs.org/" target="_blank" class="button--green"
-          >Documentation</a
-        >
-        <a
-          href="https://github.com/nuxt/nuxt.js"
-          target="_blank"
-          class="button--grey"
-          >GitHub</a
-        >
+  <div class="wrapper">
+    <header>
+      <div>
+        <logo />
+        <nav>
+          <ul>
+            <li><a href="features">Features</a></li>
+            <li><a href="team">Team</a></li>
+            <li><a href="roadmap">Roadmap</a></li>
+            <li><a href="changelog">Changelog</a></li>
+            <li class="cta"><a href="donate">Donate</a></li>
+          </ul>
+
+          <ul class="social">
+            <li>
+              <a href="https://github.com/akiraux"
+                ><img
```

Comment:
This is a weird indentation. Please avoid having the closing tags on another line

Reply:
Hey I totally agree, this is enforced by Prettier and the Dev build process, it wouldn't let me build without following these strict rules.